### ISSUE TITLE
NullPointerException datomic-toolbox.schema-resource/files

### ISSUE THREAD

**Issue by deadghost** [democracyworks/datomic-toolbox#18]
clojure 1.8.0
toolbox 2.0.4
datomic-pro 0.9.5372

```
(toolbox/initialize {:uri "datomic:mem://cmsdev" :partition :db.part/db})
 NullPointerException   datomic-toolbox.schema-resource/files (schema_resource.clj:40)
```

`lein clean` and excluding datomic-pro doesn't seem to help. I otherwise have no leads on what the issue is.

**Comment by tie-rack** [comment]
Yes. This is a bug. There is code that expects a "schemas" directory on the classpath. Until this is fixed, as a workaround, you can `mkdir resources/schemas`.

Thanks for pointing this out!

**Comment by tadler** [comment]
![blam](https://cloud.githubusercontent.com/assets/1424099/25568677/feb81fd8-2dd5-11e7-8f3e-a5e93fad32fa.gif)
